Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0010-EXTENSION-TPDM-Schemas.sql

```diff
@@ -1,8 +1,3 @@
--- SPDX-License-Identifier: Apache-2.0
```

Comment:
Losing the Ed-Fi license on these scripts.  By default MetaEd will remove these for extensions, might want to raise a ticket to handle this, it should probably generate copyright headers even on extension files if you run it in Alliance mode.  For now you'll have to just make sure they don't get removed.

Reply:
@stevenarnold-ed-fi there is already a ticket for it https://tracker.ed-fi.org/browse/METAED-1152, could you add a comment about including extensions when run in Alliance mode.

Reply:
@blmeyers  @vimayya Updated the security views and removed files related only to the ed-orgs that were removed. Also updated the claims to remove ed-orgs. Ran the EdFi.Ods.Extensions.TPDM TeamCity job on the branch and ran an initdev -UsePlugins to load the updated extension. Built and runs, all looks good.